tests/integration/Core/Framework/Increment/MySQLIncrementerTest.php

```diff
@@ -108,4 +108,43 @@ public function testReset(): void
         static::assertSame('1', $list['sw.product.index']['count']);
         static::assertSame('0', $list['sw.order.index']['count']);
     }
+
+    public function testDeleteKeys(): void
+    {
+        $this->mysqlIncrementer->increment('test-user-1', 'sw.product.index');
+        $this->mysqlIncrementer->increment('test-user-1', 'sw.product.create');
+
+        $list = $this->mysqlIncrementer->list('test-user-1');
+
+        static::assertNotEmpty($list);
+
+        $this->mysqlIncrementer->delete('test-user-1', ['sw.product.index']);
+
+        $list = $this->mysqlIncrementer->list('test-user-1');
+
+        static::assertSame([
```

Comment:
using `assertSame` on array results from the DB could be problematic as MySQL and MariaDB have different sortings